### ISSUE TITLE
Content switcher and filters buttons have incorrect padding values

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse-frontend#1362]
## Description

The paddings of Content switch buttons are incorrect on screens wider than 768px (`md` breakpoint)
First reported by @panchovm 

## Reproduction

Go to search results page, focus style of content switcher show that paddings are different.

## Screenshots

> **Note**
> This is an example and subject to change. Always refer to the Figma file for the latest design.

![image](https://user-images.githubusercontent.com/6351754/190512812-03025944-65d7-4d53-a46e-423fbdfc5355.png)

## Figma

The updated component can be found in Figma:
https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=2970%3A7554

Edited to remove the 'Filters' button. We should open a separate issue for Filters button.

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

**Comment by panchovm** [comment]
I checked the initial report and it was reviewed on a device with a view smaller than `sm` breakpoint. Now I checked again on my phone and desktop browser and both buttons look correct. However, I noticed a tiny padding detail in the large version of the component.

Top and bottom paddings in both buttons are correct. But in **Content switcher**, the padding should be `12px` for left and right sides, while in **Filters** it should be `12px` for left and `16px` for right. The bigger spacing in the right side is to balance the viewBox spacing of the 24px SVG icon.

Here a screenshot of how it should look the component's large version.

<img width="600" alt="CleanShot 2022-05-03 at 17 28 38@2x" src="https://user-images.githubusercontent.com/895819/166569511-5219d156-6d1d-4a5b-8602-3d32d86774a0.png">
